### DISCUSSION TITLE
Mask for a Entry

### DISCUSSION THREAD

**Discussion by RickWalkerOne** [TomSchimansky/CustomTkinter#1522] [Showcase]
Hello!

I'm programming in Tkinter of python, but I'm a C# programmer. Who has programmed with windows forms know that have on the TextBox (the Entry from C#) a variation with a mask, named MaskedTextbox. It's a huge help when we need that the people input a date (9999/99/99) or a number (999-999-9999). So, I create a Entry with a optional mask, if I send to you the code, you can add it on the library?

**Comment by avalon60** [comment]
This would be very useful.

**Comment by RickWalkerOne** [comment]
Hello! I made a github repository to it. It's [here](https://github.com/RickWalkerOne/CTkMaskedEntry/)

## How it seans:

![Screenshot](https://github.com/RickWalkerOne/CTkMaskedEntry/blob/main/Capturar.jpg?raw=true)
